sdk/boms/azure-sdk-bom/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.1.0 (2021-1-22)
+
+### Dependency Updates
+
+- Updated Azure SDK dependency versions to more recent releases.
+- Removed non-Azure SDK dependencies from the Azure SDK BOM.
```

Comment:
since this is a bigger change, can we link to a github issue here with more details for people who are interested?

Reply:
Sounds good.